app/views/shared/_chromebook_information_intro.html.erb

```diff
@@ -1,4 +1,7 @@
 <p class="govuk-body">
+<% if local_assigns[:show_different_devices_text] %>
```

Comment:
I wonder whether we always show this microcopy, rather than making it conditional? @fofr